### DISCUSSION TITLE
0.5.5

### DISCUSSION THREAD

**Discussion by z-huang** [z-huang/music#1335] [Announcements]
- Fix radio not working
- Playlist duplicate warning
- Add LrcLib lyrics provider
- Add marquee
- Add haptic feedback
- Black player in pure black mode

<hr />

Known issues:

- Searching lyrics will crash: disable KuGou lyrics provider for a temp fix (will be fixed in next release)

**Comment by aaru111** [comment]
revived or what?

**Comment by zippostar** [comment]
Could you include the cover as a background?

**Reply by z-huang** [reply]
What do you mean?

**Reply by zippostar** [reply]
The background is not black but the cover of the album 
![Screenshot_2024-08-10-17-08-08-394_com Arturo254 innertune](https://github.com/user-attachments/assets/2f8758ea-c6bd-420c-9abc-fd2c136a3ef9)

**Reply by z-huang** [reply]
Will consider about it.

**Comment by Snapxxx12** [comment]
please add audio fade in & fade out

**Reply by ruskcoder** [reply]
Make an issue - enhancement

**Comment by VihaanToTheRescue** [comment]
Please update backup file format to db format.
Thank you very much!

**Reply by Malopieds** [reply]
It's a zip file, unzip it and you will find the db.

**Comment by FlyingCat9013** [comment]
I have set the content region to US and it still shows my home country's content in the app. Very annoying :(. Please fix.
Thanks

**Comment by Randomboi06** [comment]
Can you please create the iOS version of this app?

**Reply by z-huang** [reply]
Too much loading. Not planned.

**Comment by EyeFrost** [comment]
Can y'all add playlist import feature, there's none. Looking forward to it cuz i cant manually get all my music in it💀

**Comment by chamudha** [comment]
Can add artist image to a round conered square please ??![Picsart_24-08-12_11-39-17-001.jpg](https://github.com/user-attachments/assets/f730e624-e8ea-404a-9227-275c9bd715d5)
